--- org.apache.commons.compress.archivers.sevenz.SevenZFile_ESTest::test0
junit.framework.AssertionFailedError: Exception was not thrown in java.io.RandomAccessFile but in org.apache.commons.compress.archivers.sevenz.SevenZFile.readHeaders(SevenZFile.java:182): org.evosuite.runtime.mock.java.lang.MockThrowable: Bad 7z signature
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at org.apache.commons.compress.archivers.sevenz.SevenZFile_ESTest.test0(SevenZFile_ESTest.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$StatementThread.run(FailOnTimeout.java:74)
--- org.apache.commons.compress.archivers.sevenz.SevenZFile_ESTest::test3
java.lang.ArrayIndexOutOfBoundsException: 1
	at org.apache.commons.compress.archivers.sevenz.SevenZFile.matches(SevenZFile.java:976)
	at org.apache.commons.compress.archivers.sevenz.SevenZFile_ESTest.test3(SevenZFile_ESTest.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$StatementThread.run(FailOnTimeout.java:74)
